v3/_locales/de/messages.json

```diff
@@ -80,6 +80,33 @@
   "popup_save_as_json": {
     "message": "Als benutzerdefinierten Auftrag speichern"
   },
+  "popup_saved": {
+    "message": "Saved!"
+  },
+  "popup_sound_1": {
+    "message": "Sound 1"
```

Comment:
Is it a mistake? To modify `de` file.

![image](https://user-images.githubusercontent.com/6510020/183300731-83136108-e847-4f22-9139-4f1e52d0b47e.png)

Reply:
No, it's just template for any German native speaker to translate these strings. I don't want to use Google translate for those languages I don't know to avoid any possible mistakes.

Reply:
OK, let's sure that this is OK for maintainers. I beleive, it's better have a non-translated string (with fallback to English, for example) rather than incorrect translations. But I'm not sure how this system (library) works.